plugin_tests/common.py

```diff
@@ -63,6 +63,12 @@ def setUp(self):
                 self.publicFolder = folder
             if folder['name'] == 'Private':
                 self.privateFolder = folder
+        folders = Folder().childFolders(self.user, 'user', user=self.admin)
+        for folder in folders:
+            if folder['name'] == 'Public':
+                self.userPublicFolder = folder
+            if folder['name'] == 'Private':
+                self.userPrivateFolder = folder
```

Comment:
Let's say I make a new account on your girder instance, and decide that the default `Public` and `Private` folders are lame and I replace them with my own.  Have I just broken this part of the code?

Reply:
This is the test where we create the folders in the test, so, no.